mmdet3d/models/detectors/groupfree3dnet.py

```diff
@@ -0,0 +1,102 @@
+import torch
+
+from mmdet3d.core import bbox3d2result, merge_aug_bboxes_3d
+from mmdet.models import DETECTORS
+from .single_stage import SingleStage3DDetector
+
+
+@DETECTORS.register_module()
+class GroupFree3DNet(SingleStage3DDetector):
```

Comment:
Just wondering is it a good idea to inherit it from votenet since the code is almost the same?

Reply:
@ZwwWayne  @Tai-Wang Not sure about the design principles regarding inheritance.

Reply:
Although the code is almost same with that of votenet especially the function get_bboxes(), basically the head of GroupFree3D is different from that of votenet. Strictly speaking, those two don't have an inheritance relationship?

Reply:
Yes, I agree they do not have an inheritance relationship. It just seems to me that this structure is not VoteNet-specific and GroupFree3D-specific either so they may be categorized as the same type. But probably this is unnecessary.

Reply:
We may refactor the structure later after this PR.